azure/services/tags/tags.go

```diff
@@ -31,7 +31,7 @@ import (
 // TagScope defines the scope interface for a tags service.
 type TagScope interface {
 	logr.Logger
-	azure.ClusterDescriber
```

Comment:
@CecileRobertMichon 
Notice - I also made a change in the `tags.go` where I changed `azure.ClusterDescriber` to `azure.Authorizer` as we weren't using any `azure.ClusterDescriber` interface methods in `tags.go`

Reply:
good catch, thanks for changing it